internal/node/model/models.go

```diff
@@ -10,47 +10,90 @@ import (
 
 type Hash = common.Hash
 type Address = common.Address
+type Status string
+
+const (
+	None                       Status = "NONE"
```

Comment:
It would better be named `InputStatus` instead of just `Status`  as these are just for Inputs

Maybe they should be defined on its own module, so when it is used outside, it would be `inputstatus.None` instead of `model.None`. Or it should be prefixed with "InputStatus" becaming `model.InputStatusNone`

Reply:
Actually it can be renamed to the one used at DB definition `InputCompletionStatus` , as it will improve future maintainability and reading